packages/wallet/api/src/lib-wallet.js

```diff
@@ -451,6 +451,40 @@ export function makeWalletRoot({
     return proposalForDisplay;
   };
 
+  /**
+   * Delete inactive offers.
```

Comment:
```suggestion
   * Delete offer once inactive.
```
Maybe also worth explaining that this is idempotent. It can (and will?) be called multiple times for the same id.